Library/Homebrew/utils/pypi.rb

```diff
@@ -182,7 +182,7 @@ def update_python_resources!(formula, version: nil, package_name: nil, extra_pac
 
     extra_packages = (extra_packages || []).map { |p| Package.new p }
     exclude_packages = (exclude_packages || []).map { |p| Package.new p }
-    exclude_packages += %W[#{main_package.name} argparse pip setuptools wheel wsgiref].map { |p| Package.new p }
+    exclude_packages += %W[#{main_package.name} argparse pip setuptools wsgiref].map { |p| Package.new p }
```

Comment:
A comment above this explaining why these are excluded might be helpful.

Reply:
Good idea. The problem is that I don't know why half of them (`argparse`, `wsgiref`) are in the list. When I wrote this, I'm sure I copied that list from somewhere else (otherwise I wouldn't have come up with those) but I can't seem to find where that list is.